Tests/test_imageops.py

```diff
@@ -311,3 +313,32 @@ def autocontrast(cutoff):
 
         assert autocontrast(10) == autocontrast((10, 10))
         assert autocontrast(10) != autocontrast((1, 10))
+
+
+def test_autocontrast_mask_toy_input():
+    # Test the mask argument of autocontrast
+    with Image.open("Tests/images/bw_gradient.png") as img:
+
+        rect_mask = Image.new("L", img.size, 0)
+        draw = ImageDraw.Draw(rect_mask)
+        x0, y0 = img.size[0] // 4, img.size[1] // 4
+        x1, y1 = 3 * img.size[0] // 4, 3 * img.size[1] // 4
```

Comment:
Let's also put these on their own lines for clarity:

```suggestion
        x0 = img.size[0] // 4
        y0 = img.size[1] // 4
        x1 = 3 * img.size[0] // 4
        y1 = 3 * img.size[1] // 4
```